### ISSUE TITLE
label:enhancement  HomeAssistant friendly url's

### ISSUE THREAD

**Issue by maphilli14** [aaronwmorris/indi-allsky#203]
I'm a big HASS user and would love to get the timelapse  view embeddable into lovelace in HASS.  Currently using the

`https://allsky/indi-allsky/videoviewer`

has the side bar and looks wonky as a card...

![image](https://user-images.githubusercontent.com/6034105/170292427-491d820d-4032-42d3-8df3-b9906a5d059f.png)

**Comment by maphilli14** [comment]
Additionally I've tried using some custom py to rename the date outputs to a static filename for easy linking -



```
#! /usr/bin/python3

import os,shutil

srcdir='/var/www/html/Astronomy/allsky'

'''
## 2022.05.24

Written by some google fu by Michael A. Phillips
The idea is that indi-allsky does a file upload and then does copy of date to latest for static 
file embedding into home assistant

## 2022.05.24

https://www.geeksforgeeks.org/python-finding-strings-with-given-substring-in-list/

# Python code to demonstrate
# to find strings with substrings
# using list comprehension

# initializing list
test_list = ['GeeksforGeeks', 'Geeky', 'Computers', 'Algorithms']

# printing original list
print ("The original list is : " + str(test_list))

# initializing substring
subs = 'Geek'

# using list comprehension
# to get string with substring
res = [i for i in test_list if subs in i]

# printing result
print ("All strings with given substring are : " + str(res))

'''
k='allsky-keogram'
s='allsky-startrail'
t='allsky-timelapse'
K=sorted(os.listdir(srcdir))

lK = [i for i in K if k in i]
lK1=lK[-1]
lK2=lK[-2]
shutil.copy(os.path.join(srcdir,lK1),os.path.join(srcdir,'latestkeogram1.jpg'))
shutil.copy(os.path.join(srcdir,lK2),os.path.join(srcdir,'latestkeogram2.jpg'))

lS = [i for i in K if s in i]
lS=lS[-1]
shutil.copy(os.path.join(srcdir,lS),os.path.join(srcdir,'lateststartrail.jpg'))

lT = [i for i in K if t in i]
lT1=lK[-1]
lT2=lK[-2]
shutil.copy(os.path.join(srcdir,lT1),os.path.join(srcdir,'latesttimelapse1.mp4'))
shutil.copy(os.path.join(srcdir,lT1),os.path.join(srcdir,'latesttimelapse2.mp4'))

```

**Comment by aaronwmorris** [comment]
That screenshot is interesting.  It looks like two pages were overlayed with each other.

**Comment by maphilli14** [comment]
I think it just rendered the side bar across the top instead of side.   It works, but I'd like it without and perhaps somewhat customizable but I'm not the well versed in MQTT as an option or lovelace cards.  Best I could come up with was using latest-blah instead of datexyz-blah.

**Comment by aaronwmorris** [comment]
First let me say that web design and CSS are not my strongest areas.  :-)

The template I used will place the navigation bar on the top once the horizontal size of the screen is less than 576px.  You can test this by resizing you browser window.  Perhaps if I remove the status, sun, and moon information when it adjusts it will look cleaner.  I will run some tests.

**Comment by aaronwmorris** [comment]
I merged #207 to remove the status information when the page is resized.  Let me know if that is better.

**Comment by aaronwmorris** [comment]
I am going to close this issue for now.  Let me know if there is anything else I can do for you.
